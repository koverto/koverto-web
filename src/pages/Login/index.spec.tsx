import * as React from "react"
import { render } from "@testing-library/react"
import { Login } from "."

describe("Login", () => {
  it("matches the snapshot", () => {
    const { container } = render(<Login />)
    expect(container).toMatchSnapshot()
  })
})
