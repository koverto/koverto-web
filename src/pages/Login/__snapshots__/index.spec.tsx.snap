// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login matches the snapshot 1`] = `
<div>
  <h1>
    Login
  </h1>
</div>
`;
